### PR TITLE
Added Toad_Hollow

### LINKED ISSUE #10: Add Local Business Toad Hollow Nursery
https://www.toadhollownursery.com/

### PR DESCRIPTION
**Pull request by Yashh56**
Fixes #10

### CODE REVIEW

File: businesses/Toad_Hollow_Nursery/index.qmd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Toad Hollow"
+categories: [Nursery]
```
Comment (myelinviolin) [inline businesses/Toad_Hollow_Nursery/index.qmd]: change category to "Gardening"

File: businesses/Toad_Hollow_Nursery/index.qmd [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+description: Local Nursery and Gardern
```
Comment (myelinviolin) [inline businesses/Toad_Hollow_Nursery/index.qmd]: Change Description to "Local Native Plant Nursery"

File: businesses/Toad_Hollow_Nursery/leaf.webp
Comment (myelinviolin) [inline businesses/Toad_Hollow_Nursery/leaf.webp:1]: Please change to this [logo](https://static.wixstatic.com/media/d3de1a_72a1a50a713d43f6b35e84892909d399~mv2.png/v1/crop/x_312,y_223,w_894,h_1304/fill/w_96,h_140,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/3_18%20picture%20logo%20only%20-%20Cropped%20smaller.png)

File: businesses/Toad_Hollow_Nursery/index.qmd
Comment (myelinviolin) [inline businesses/Toad_Hollow_Nursery/index.qmd:1]: Please add their business description from the webpage so it is not all images. People need to know what they are about just by looking at this page.

**Review by myelinviolin — CHANGES_REQUESTED** [review]
I think I have to leave this comment here so the review is complete so it becomes your turn to address the comments.

File: businesses/Toad_Hollow_Nursery/index.qmd
Comment (Yashh56) [reply businesses/Toad_Hollow_Nursery/index.qmd:1]: Only Description ?

**Comment by Yashh56** [comment]
I will make these changes ASAP !!

**Review by myelinviolin — APPROVED** [review]
closes #10

**Pull request merged**